### PR TITLE
Add tone

### PR DESCRIPTION
**Pull request by peraneko**
## Description
Add tone keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/peranekofactory/tone/rev1/config.h [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ * 
+ */
+
+#define MATRIX_ROW_PINS { NO_PIN }
+#define MATRIX_COL_PINS { F6,F7,B1,B3,D4,C6,D7,E6 }
```
Comment (drashna) [inline keyboards/peranekofactory/tone/rev1/config.h:45]: You should use direct pins them. 

https://docs.qmk.fm/#/hardware_avr?id=direct-pin-matrix

File: keyboards/peranekofactory/tone/rev2/config.h [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ * 
+ */
+
+#define MATRIX_ROW_PINS { NO_PIN }
+#define MATRIX_COL_PINS { D4,C6,D7,E6,F6,F7,B1,B3 }
```
Comment (drashna) [inline keyboards/peranekofactory/tone/rev2/config.h:45]: Also should be using direct pins.

File: keyboards/peranekofactory/tone/rev1/config.h (hunk 1 above)
Comment (peraneko) [reply keyboards/peranekofactory/tone/rev1/config.h:51]: Changed to use Direct_Pins in the relevant part.
It has been confirmed to work on the actual machine.

File: keyboards/peranekofactory/tone/rev2/config.h (hunk 2 above)
Comment (peraneko) [reply keyboards/peranekofactory/tone/rev2/config.h:45]: Changed to use Direct_Pins in the relevant part.
It has been confirmed to work on the actual machine.

**Review by drashna — APPROVED** [review]

File: keyboards/peranekofactory/tone/readme.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+TONE is macro_pad With RotaryEncoder.
+
+* Keyboard Maintainer: [peraneko](https://github.com/peraneko)
+* Hardware Supported: TONE
+* Hardware Availability: https://shop.yushakobo.jp/collections/keyboard-1/products/consign_tone-note
+
+Make example for this keyboard (after setting up your build environment):
+
+    make peranekofactory/tone:default
```
Comment (fauxpark) [inline keyboards/peranekofactory/tone/readme.md:13]:
```suggestion
    make peranekofactory/tone:default

Flashing example for this keyboard:

    make peranekofactory/tone:default:flash

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/peranekofactory/tone/rev1/readme.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+TONE is macro_pad With RotaryEncoder.
+
+* Keyboard Maintainer: [peraneko](https://github.com/peraneko)
+* Hardware Supported: TONE
+* Hardware Availability: https://shop.yushakobo.jp/collections/keyboard-1/products/consign_tone-note
+
+Make example for this keyboard (after setting up your build environment):
+
+    make peranekofactory/tone/rev1:default
```
Comment (fauxpark) [inline keyboards/peranekofactory/tone/rev1/readme.md:13]:
```suggestion
    make peranekofactory/tone/rev1:default

Flashing example for this keyboard:

    make peranekofactory/tone:default:flash

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/peranekofactory/tone/rev2/readme.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+TONE_rev2 is macro_pad With RotaryEncoder.
+
+* Keyboard Maintainer: [peraneko](https://github.com/peraneko)
+* Hardware Supported: TONE
+* Hardware Availability: https://shop.yushakobo.jp/collections/keyboard-1/products/consign_tone-note
+
+Make example for this keyboard (after setting up your build environment):
+
+    make peranekofactory/tone/rev2:default
```
Comment (fauxpark) [inline keyboards/peranekofactory/tone/rev2/readme.md:13]:
```suggestion
    make peranekofactory/tone/rev2:default

Flashing example for this keyboard:

    make peranekofactory/tone/rev2:default:flash

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by peraneko** [comment]
I'm sure you're busy, but it's going to be a new keyboard, so please give us a review of the other one.

File: keyboards/peranekofactory/tone/rules.mk [hunk 6]
```diff
@@ -0,0 +1 @@
+DEFAULT_FOLDER = tone/rev2
```
Comment (mtei) [inline keyboards/peranekofactory/tone/rules.mk:1]:
```suggestion
DEFAULT_FOLDER = peranekofactory/tone/rev2
```

**Review by mtei — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**